config.xml

```diff
@@ -60,9 +60,11 @@
                 <category android:name="android.intent.category.BROWSABLE" />
                 <data android:scheme="https" android:host="s3.amazonaws.com" android:path="/outline-vpn/invite.html" />
             </intent-filter>
+            <!-- Disable Firebase Analytics data collection for beta builds. -->
```

Comment:
I think you mean "disable for production builds"?

Reply:
I do mean beta :) Non-beta (production and debug) builds don't link Crashlytics/Firebase, so these options are not used.